ftw/contacts/contents/contact.py

```diff
@@ -0,0 +1,236 @@
+from collective import dexteritytextindexer
+from ftw.contacts import _
+from ftw.contacts.utils import get_contact_title
+from plone.app.textfield import RichText
+from plone.app.textfield.value import RichTextValue
+from plone.autoform import directives as form
+from plone.dexterity.content import Item
+from plone.namedfile.field import NamedBlobImage
+from plone.supermodel import model
+from z3c.schema.email import RFC822MailAddress
+from zope import schema
+from zope.interface import Invalid
+from zope.interface import invariant
+from zope.schema.vocabulary import SimpleTerm
+from zope.schema.vocabulary import SimpleVocabulary
+
+
+class NoOrganizationOrFullname(Invalid):
+    """The organization or fullname is invalid
+    """
+
+
+gender_choice_vocabulary = SimpleVocabulary([
+    SimpleTerm(value=u'm', title=_(u'label_male', default=u'Male')),
+    SimpleTerm(value=u'f', title=_(u'label_female', default=u'Female')),
+    SimpleTerm(value=u'', title=_(u'label_gender_neutral', default=u'-'))
+    ])
+
+
+class IContact(model.Schema):
```

Comment:
We usually do not use the schema interface as "primary" interface of the type.
So to be consistent with other ftw-packages a propose these changes:
- rename `IContact` to `IContactSchema`
- add a marker interface `IContact` to the interfaces.py (pkg root)
- add `implements(IContact)` to the `Contact` class
- do the same for the other types